app/ui/window.js

```diff
@@ -93,6 +93,7 @@ module.exports = class Window {
           cwd: process.argv[1] && isAbsolute(process.argv[1]) ? process.argv[1] : cfgDir,
           splitDirection: undefined,
           shell: cfg.shell,
+          isWsl: /(?:bash|debian|kali|opensuse|ubuntu|wsl)\.exe$/.test(cfg.shell),
```

Comment:
I believe there is also `ubuntu1804.exe`

Reply:
I must admit I added these by intuition because I couldn't find a document with all the distros exectuables. I only use Debian myself. I'll add that version to the list soon.

However, as the number of distros may grow and some of them may have multiple executables (case in point), a regex is probably not the best approach. But I honestly can't think of another way to detect if Hyper is running a WSL shell.

Any other ideas?

Reply:
Yes actually :smile: 

I only know about that exe because I recently did something very similar in vscode. Check out the list I came up with https://github.com/Microsoft/vscode/blob/86adb527f12c12c1c04806cfa0cd435f98475a6f/src/vs/workbench/parts/terminal/node/windowsShellHelper.ts#L13 (which also includes powershell and cmd, everything else is wsl).

Reply:
Based on your list, I guess I have a couple of entries to fix 😄 

I might follow your approach instead of regex, it will be easier to maintain in the long run.

Reply:
@Tyriar Changed the implementation, could you please re-review? I've also took the liberty of using `path.sep` instead of the hard-coded `/` in one other instance.

Reply:
I don't know the code base too well so can't give a full review, the list of exes looks good though :+1: